[BUG] test_correct_image_brightness sometimes fails locally on Windows

**Describe the bug**
`test_correct_image_brightness` sometimes fails locally when I run `pytest` from the terminal.

Weirdly `test_correct_image_brightness` has not failed for me (yet?) in the VS code testsuite using the same environment (see screenshot) or (less surprisingly) in GitHub Actions.

@K-Meech and I did some troubleshooting but could not figure out the issue.

The issue persists even after I:
- restart VS Code
- reinstall the conda env

Things we noted:
-  Whenever test fails assertion error seems the same (AssertionError: assert 0.07657468157481162 < 0.04)
- I have the same version of `simpleitk` installed locally as @K-Meech who does not have the issue (`2.5.2`)
- The test date is where it is supposed to be in `"~/.brainglobe/brainglobe-workflows/test-data" `

**To Reproduce**
- Clone [brainglobe-template-builder](https://github.com/brainglobe/brainglobe-template-builder) and install within a `conda` environment using the recommended instructions (see below).

```
conda env create -n template-builder -f environment.yaml
conda activate template-builder

git clone https://github.com/brainglobe/brainglobe-template-builder
cd brainglobe-template-builder

pip install -e .[dev]
```

run ptyest from the terminal
```
pytest
```

**Expected behaviour**
Sometimes `test_correct_image_brightness` fails locally, other times not. 


**Log file**

> 
> ```
> FAILED tests/test_unit/test_brightness.py::test_correct_image_brightness - AssertionError: assert 0.07657468157481162 < 0.04
> ```

**Screenshots**

<img width="1916" height="1030" alt="Image" src="https://github.com/user-attachments/assets/59103e44-96d5-4ced-b7a3-8e1eddc9c354" />

**Computer used (please complete the following information):**

> OS Name	Microsoft Windows 11 Pro
> Version	10.0.26200 Build 26200
> OS Manufacturer	Microsoft Corporation
> System Manufacturer	Dell Inc.
> System Model	Latitude 5440
> System Type	x64-based PC
> Processor	13th Gen Intel(R) Core(TM) i7-1365U, 1800 Mhz, 10 Core(s), 12 Logical Processor(s)
> 

**Additional context**

--- a/tests/test_unit/test_brightness.py
+++ b/tests/test_unit/test_brightness.py
@@ -18,9 +18,6 @@ def test_correct_image_brightness():
     ants_corrected_image = load_any(corrected_image_path())
 
     assert output_image.shape == ants_corrected_image.shape
-    # Check largest pixel value difference between ants correction and
-    # our own is 0.04
-    # Note: the github actions windows runner needs this higher 0.04
-    # threshold, but all others (as well as running locally on a windows
-    # computer), can use a tighter 0.01 threshold
-    assert np.absolute(ants_corrected_image - output_image).max() < 0.04
+    # Note: max difference values can differ slightly across platforms, but
+    # the max relative difference should be below 1E-3 in all cases.
+    np.testing.assert_allclose(output_image, ants_corrected_image, rtol=1e-03)

